package.json

```diff
@@ -114,7 +115,8 @@
     "rollup:snooze-bootstrap": "rollup -c --environment entry:popup/snooze-bootstrap",
     "rollup:confirm-bar": "rollup -c --environment entry:lib/confirm-bar",
     "bundle:css": "npm run lint:sass && shx mkdir -p dist/popup && node-sass src/popup/snooze.scss > dist/popup/snooze.css",
-    "package": "npm run build && web-ext build -s dist && mv web-ext-artifacts/*.zip addon.xpi",
+    "package": "cross-env NODE_ENV=production npm run build && web-ext build -s dist && mv web-ext-artifacts/*.zip addon.xpi",
+    "package:dev": "cross-env NODE_ENV=development npm run build && web-ext build -s dist && mv web-ext-artifacts/*.zip addon.xpi",
```

Comment:
Should we move this to `addon-dev.xpi` instead, to help avoid sending the wrong xpi?